components/brave_new_tab_ui/components/default/braveToday/customize/Configure.tsx

```diff
@@ -109,12 +108,29 @@ const Content = styled.div`
 `
 
 export default function Configure () {
-  const [enabled, setEnabled] = useNewTabPref('isBraveTodayOptedIn')
-  const { setCustomizePage, customizePage } = useBraveNews()
+  const {
+    setCustomizePage,
+    customizePage,
+    setOptInPrefEnabled,
+    setShowOnNTPPrefEnabled,
+    optInPrefEnabled,
+    showOnNTPPrefEnabled
+  } = useBraveNews()
+
+  const handleEnabledChange = (shouldEnable: boolean) => {
+    if (shouldEnable) {
+      setOptInPrefEnabled(true)
+      setShowOnNTPPrefEnabled(true)
+      return
+    }
+    setShowOnNTPPrefEnabled(false)
```

Comment:
So there's no way to opt out?

Reply:
correct. The opt-in just means that you've seen the terms, really. You can still turn the feature off which will disable all the fetching.